samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/CMakeLists.txt

```diff
@@ -1,6 +1,9 @@
 cmake_minimum_required(VERSION 3.13.1)
 set(QEMU_EXTRA_FLAGS -s)
 
+set(CONF_FILE prj.conf)
+#list(APPEND CONF_FILE prj_smp_svr.conf)
```

Comment:
This looks like a debug line which is not needed.

My point is that the user can use `-DOVERLAY_CONFIG=prj_smp_svr.conf` at cmake time to mix in a fragment which will make the application work under MCUboot and bring the OTA update code into the build.